setup.py

```diff
@@ -104,37 +104,37 @@ def default_setup_args(*, version):
     return setup_args
 
 
-version = "0.3.1"
+version = "0.4.0"
 version = update_version(version, use_file_if_exists=False, create_file=True)
 
 install_requires = [
     # common module provides utils with stable api across minor version
-    "autogluon.common>=0.7,<1.1",
+    "autogluon.common>=0.7",
     # <2 because unlikely to introduce breaking changes in minor releases. >=1.10 because 1.10 is 3 years old, no need to support older
     "boto3>=1.10,<2.0",
-    "numpy>=1.21,<1.27",
+    "numpy>=1.21,<1.29",
```

Comment:
We don't need this as core comes with numpy dependency: https://github.com/autogluon/autogluon/blob/master/core/src/autogluon/core/_setup_utils.py#L20

Reply:
Good point. Will update!